easybuild/easyconfigs/s/Scalene/Scalene-1.5.20-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'Scalene'
+version = '1.5.20'
+
+homepage = 'https://github.com/plasma-umass/scalene'
+description = """Scalene is a high-performance CPU, GPU and memory profiler for Python that does a number of things
+that other Python profilers do not and cannot do. It runs orders of magnitude faster than other profilers while
+delivering far more detailed information."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('cloudpickle', '2.2.1', {
+        'checksums': ['d89684b8de9e34a2a43b3460fbca07d09d6e25ce858df4d5a44240403b6178f5'],
+    }),
+    ('pynvml', '11.4.1', {
+        'checksums': ['b2e4a33b80569d093b513f5804db0c7f40cfc86f15a013ae7a8e99c5e175d5dd'],
+    }),
+    ('commonmark', '0.9.1', {
+        'checksums': ['452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60'],
+    }),
+    ('rich', '13.1.0', {
+        'checksums': ['81c73a30b144bbcdedc13f4ea0b6ffd7fdc3b0d3cc259a9402309c8e4aee1964'],
+    }),
+    ('astunparse', '1.6.3', {
```

Comment:
is this really needed? seems only required for Python < 3.9
https://github.com/plasma-umass/scalene/blob/2814d8c11b28ba480988eedbf1617af7a925c174/setup.py#L172C1-L172C54

Reply:
The setup.py requirements file was different in 1.5.20 release
https://github.com/plasma-umass/scalene/blob/033a9c6129ab6cc06d309e8bd3d5a8bd0d3fe3ba/setup.py#L205
but maybe they were wrong